src/qml/filters/gpstext/ui.qml

```diff
@@ -31,12 +31,13 @@ Item {
     
     property url settingsOpenPath: 'file:///' + settings.openPath
     Shotcut.File { id: gpsFile }
+    property int js_tz_offset: 0
     
     signal fileOpened(string path)
     onFileOpened: settings.openPath = path
     
     Component.onCompleted: {
-        var resource = filter.get('gps.file')
+        var resource = filter.get('resource')
```

Comment:
For backwards compatibility, you should check for "gps.file" if "resource" returns an empty string. Else, existing projects will break when they are opened with this new version.